package.json

```
@@ -1,125 +1,125 @@
+    "eject": "expo eject",
+    "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\" && eslint \"{src,apps,libs,test}/**/*.{js,jsx,ts,tsx}\" --fix",
+    "format:check": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\" && eslint \"{src,apps,libs,test}/**/*.{js,jsx,ts,tsx}\" --max-warnings=0",
+    "type:check": "tsc",
+    "grepI18n": "node src/Common/Hooks/i18n-js/grepI18n.ts"
+  },
+  "dependencies": {
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
+    "@ctrl/react-adsense": "^1.6.1",
+    "@expo-google-fonts/lato": "^0.2.3",
+    "@expo-google-fonts/montserrat": "^0.2.3",
+    "@expo-google-fonts/noto-sans-javanese": "^0.2.3",
+    "@expo-google-fonts/open-sans": "^0.2.3",
+    "@expo-google-fonts/oswald": "^0.2.3",
+    "@expo-google-fonts/poppins": "^0.2.3",
+    "@expo-google-fonts/roboto": "^0.2.3",
+    "@expo-google-fonts/roboto-condensed": "^0.2.3",
+    "@expo-google-fonts/roboto-mono": "^0.2.3",
+    "@expo-google-fonts/source-sans-pro": "^0.2.3",
+    "@googlemaps/google-maps-services-js": "^3.3.16",
+    "@gorhom/bottom-sheet": "^4.4.5",
+    "@react-google-maps/api": "^2.13.1",
+    "@react-native-async-storage/async-storage": "1.18.2",
+    "@react-native-community/datetimepicker": "7.2.0",
+    "@react-native-community/slider": "4.4.2",
+    "@react-native-firebase/analytics": "^17.5.0",
+    "@react-native-firebase/app": "^17.5.0",
+    "@react-native-firebase/crashlytics": "^17.5.0",
+    "@react-native-firebase/dynamic-links": "^17.5.0",
+    "@react-native-firebase/perf": "^17.5.0",
+    "@react-native-masked-view/masked-view": "0.2.9",
+    "@react-navigation/material-bottom-tabs": "^6.2.4",
+    "@react-navigation/material-top-tabs": "^6.3.1",
+    "@react-navigation/native": "^6.0.13",
+    "@react-navigation/native-stack": "^6.9.1",
+    "expo": "^49.0.0",
+    "expo-build-properties": "~0.8.3",
+    "expo-clipboard": "~4.3.1",
+    "expo-dev-client": "~2.4.11",
+    "expo-font": "~11.4.0",
+    "expo-image-manipulator": "~11.3.0",
+    "expo-image-picker": "~14.3.2",
+    "expo-linking": "~5.0.2",
+    "expo-localization": "~14.3.0",
+    "expo-status-bar": "~1.6.0",
+    "firebase": "9.15.0",
+    "i18n-js": "4.2.3",
+    "react": "18.2.0",
+    "react-datepicker": "^4.8.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.72.5",
+    "react-native-autocomplete-input": "^5.2.0",
+    "react-native-drax": "^0.10.3",
+    "react-native-gesture-handler": "~2.12.0",
+    "react-native-google-mobile-ads": "^9.1.1",
+    "react-native-maps": "1.7.1",
+    "react-native-pager-view": "6.2.0",
+    "react-native-paper": "^4.12.5",
+    "react-native-popup-menu": "^0.16.1",
+    "react-native-ratings": "^8.1.0",
+    "react-native-reanimated": "~3.3.0",
+    "react-native-safe-area-context": "4.6.3",
+    "react-native-screens": "~3.22.0",
+    "react-native-tab-view": "^3.3.4",
+    "react-native-uuid": "^2.0.1",
+    "react-native-vector-icons": "^9.2.0",
+    "react-native-view-shot": "3.7.0",
```

react native view shot は、独自実装しているから変更しないで欲しい！

```suggestion
    "react-native-view-shot": "github:Ayato-kosaka/react-native-view-shot-with-web-support#master",
```